src/model_theory/terms_and_formulas.lean

```diff
@@ -380,38 +578,54 @@ def semantically_equivalent_setoid (T : L.Theory) : setoid (L.bounded_formula α
   iseqv := ⟨λ φ M ne str v xs hM, by simp,
     λ φ ψ h M ne str v xs hM, begin
       haveI := ne,
-      have h := h M v xs hM,
-      rw [realize_inf, and_comm] at h,
-      rw realize_inf,
-      exact h,
+      rw [realize_bd_iff, iff.comm, ← realize_bd_iff],
+      exact h M v xs hM,
     end, λ φ ψ θ h1 h2 M ne str v xs hM, begin
       haveI := ne,
       have h1' := h1 M v xs hM,
       have h2' := h2 M v xs hM,
-      rw [realize_inf, realize_imp, realize_imp] at *,
+      rw [realize_bd_iff] at *,
       exact ⟨h2'.1 ∘ h1'.1, h1'.2 ∘ h2'.2⟩,
     end⟩ }
 
-lemma semantically_equivalent_not_not {T : L.Theory} {φ : L.bounded_formula α n} :
+lemma semantically_equivalent_bd_not_bd_not {T : L.Theory} {φ : L.bounded_formula α n} :
```

Comment:
With my other comment, I would make this `bounded_formula.semantically_equivalent_not_not`, and make `φ` explicit.